lib/modules/genre.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../item'
+
+class Genre < Item
```

Comment:
- [ ] Great work building your Genre class 💯. However, please kindly double-check if Genre inherits from Item `Genre < Item`, in the UML provided the relationship between Genre and Item seems to be one-to-many and not inheritance 👇🏽.

![cr1](https://user-images.githubusercontent.com/116701485/234328608-39327cd3-e59a-4e78-bfcc-000477a73472.png)